package.json

```diff
@@ -72,7 +73,7 @@
     "protractor": "5.4.1",
     "ts-node": "^7.0.1",
     "tslint": "5.11.0",
-    "typescript": "^3.1.1",
+    "typescript": "^3.2.4",
```

Comment:
Can remove the changes to `package.json`?

Reply:
Sure.